planning/backlog.md

```diff
@@ -0,0 +1,7 @@
+## Could-Haves
+
+would be really cool ... if there's time:
+- As a **user** I want to get an opinion where is the location of office so that I add map information.
+  - [ ] Given live map context when trying to know where it is  then searching on map.
```

Comment:
"Given the user tries to know where the hack your future office is, it should be a live map for searching and adding on the map" 

can you change this line with the sentence above like this?